site/reference/pkg/init/README.md

```diff
@@ -3,31 +3,43 @@ title: "Init"
 linkTitle: "init"
 type: docs
 description: >
-   Initialize an empty package
+   Initialize an empty package.
 ---
 <!--mdtogo:Short
-    Initialize an empty package
+    Initialize an empty package.
 -->
 
-{{< asciinema key="pkg-init" rows="10" preload="1" >}}
+`init` initializes an existing empty directory as an empty kpt package.
 
-Init initializes an existing empty directory as an empty kpt package.
+### Synopsis
+<!--mdtogo:Long-->
+```
+kpt pkg init [DIR] [flags]
+```
+
+#### Args
+
+```
+DIR:
+  init fails if DIR does not already exist. Defaults to the current working directory.
+```
 
-**init is optional**: Any directory containing Kubernetes Resource
-Configuration may be treated as remote package without the existence of
-additional packaging metadata.
+#### Flags
 
-* Resource Configuration may be placed anywhere under DIR as *.yaml files.
-* DIR may contain additional non-Resource Configuration files.
-* DIR must be pushed to a git repo or repo subdirectory.
+```
+--description
+  Short description of the package. (default "sample description")
 
-Init will augment an existing local directory with packaging metadata to help
-with discovery.
+--name
+  The package name. Defaults to the directory base name.
```

Comment:
Not sure this flag makes sense, given`metadata.name` must match the directory name.

Reply:
Agree. I have removed the flag.